src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilteredBy } from '../../types/FilteredBy';
+
+type Props = {
+  todos: Todo[],
+  filteredBy: string,
+  setFilteredBy: (value: FilteredBy) => void,
+  handleDeleteTodo: (todoId: number) => void,
+};
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  filteredBy,
+  setFilteredBy,
+  handleDeleteTodo,
+}) => {
+  const completedTodos = todos.filter(todo => todo.completed);
+  const handleClearCompleted = () => {
+    completedTodos.forEach(todo => {
+      handleDeleteTodo(todo.id);
+    });
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todos.length} items left`}
```

Comment:
please change the number to correct one. should be the number of **uncompleted** todos